### PR TITLE
placeholder added

### PR DESCRIPTION
**Pull request by anassalam**
placeholder added to input type text

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md [hunk 1]
```diff
@@ -71,7 +71,31 @@ tests:
...
 <section id='solution'>
 
-```js
-// solution required
+<div id='html-seed'>
+
+```html
+<h2>CatPhotoApp</h2>
+<main>
+  <p>Click here to view more <a href="#">cat photos</a>.</p>
```
Comment (itaisteinherz) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md]:
```suggestion
  <p><a href="#">Click here</a> to view more cat photos.</p>
```

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md [hunk 2]
```diff
@@ -71,7 +71,31 @@ tests:
...
+
+  <a href="#"><img src="https://bit.ly/fcc-relaxing-cat" alt="A cute orange cat lying on its back."></a>
+
+  <p>Things cats love:</p>
+  <ul>
+    <li>cat nip</li>
```
Comment (itaisteinherz) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md]: You should capitalize the first letter of the first word of each list item.

**Review by itaisteinherz — CHANGES_REQUESTED** [review]
This looks great! I commented a couple of suggestions related to meta changes.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md]: @itaisteinherz It is just a list, so it is not necessary to capitalize.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md [hunk 3]
```diff
@@ -71,7 +71,31 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
-// solution required
+<div id='html-seed'>
```
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md]: Line 74 should not be a part of the solution.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md [hunk 4]
```diff
@@ -71,7 +71,31 @@ tests:
...
+  <ol>
+    <li>flea treatment</li>
+    <li>thunder</li>
+    <li>other cats</li>
+  </ol>
+  <input type="text" placeholder="cat photo URL">
+</main>
 ```
+
+</div>
```
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.english.md]: Line 99 should not be a part of the solution.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Please see my comments regarding changes needed.

**Comment by RandellDawson** [comment]
Hi @anassalam 

As there has been no response from your end for a while, I am closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**